tests/test_uritemplate.rs

```diff
@@ -1,5 +1,5 @@
-extern crate uritemplate;
-use uritemplate::{IntoTemplateVar, TemplateVar, UriTemplate};
+extern crate datta;
```

Comment:
is it needed?

Reply:
I still need to migrate to 2021 edition :grin: